src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -21,8 +21,10 @@ int gd_dev_apogee_sp215_read(void){
         value += Wire.read();
     }
     Wire.endTransmission();
-    /* Analog to digital conversion with 12 bit-resolution */   
-    value = (value*5000.00)/(0xFFF);
+    /* Analog to digital conversion with 16 bit-resolution */   
+    /* Divide by 0xFFFF to convert 16-bit ADC reading, 5 V is reference voltage */
+    /* Physical voltage divider implemented to prevent saturation of signal. Apogee SP215 has a maximum output voltage of 5V but the MCP3425 only allows a max input of 2.048V. Voltage divider over .973kohm and 2.167kohm resistor used to decrease amplitude of signal from Apogee. Use a scaling factor of 3.22713257965 to account for this voltage divider and calculate the value of the original signal. */
```

Comment:
This comment is still hard to read, I would probably break it up more than it already is. You're already using mostly complete sentences, so you can just convert everything to complete sentences to be more clear. 

Also, can you use variables to make things more clear?